.gitignore

```diff
@@ -25,3 +25,9 @@ vendor/
 
 node_modules/
 web/static/
+
+# .DS_Store
+.DS_Store
+web/.DS_Store
```

Comment:
You should just be able to put `.DS_Store` like on line 31 and it will ignore it in all directories